nodejs/src/Render.js

```diff
@@ -242,6 +242,18 @@ const Render = (input, SETTINGS = {}, debug = DEBUG.enabled, debuglevel = DEBUG.
 			`</div>`;
 	}
 
+	// handle raw mode
+	if (OPTIONS.env === 'node') {
+		if (OPTIONS.rawMode === true) {
+			const re = /\n/g;
+			const newline = '\r\n';
+			write = write.replace(re, newline);
+			output = output.map((x) => {
+				return x.replace(re, newline);
+			});
+		}
+	}
```

Comment:
Instead of replacing after-the-fact, maybe add a variable at the top that defaults to `\n` and is changed with the option later to `\r\n`?

Reply:
Can do, but it would require a bit more refactoring to the logic. I would want the OK from you if you think we should make those changes.
See #79 for more details.